api/src/pcapi/routes/public/individual_offers/v1/endpoints.py

```diff
@@ -86,6 +87,12 @@ def post_product_offer(
                 venue=venue,
                 visual_disability_compliant=body.disability_compliance.visual_disability_compliant,
             )
+            if body.stock:
+                offers_api.create_stock(
+                    offer=offer,
+                    price=decimal.Decimal(body.stock.price) / 100,
```

Comment:
(détail) Ton code est bon, mais je trouve que c'est une bonne habitude que d'utiliser les fonctions de `finance.utils` (notamment la conversion inverse, d'euros vers centimes, qui n'est pas si évidente).

```suggestion
                    price=core.finance.utils.to_euros(body.stock.price),  # cents -> euros
```